market-generator/src/main/java/com/market/banica/generator/exception/NoSuchGoodException.java

```diff
@@ -0,0 +1,7 @@
+package com.market.banica.generator.exception;
+
+public class NoSuchGoodException extends RuntimeException {
```

Comment:
I would suggest to create more generic name of the exception to handle more cases - the message could specify that the good is not found